internal/formats/syftjson/model/document.go

```diff
@@ -4,10 +4,12 @@ package model
 type Document struct {
 	Artifacts             []Package      `json:"artifacts"` // Artifacts is the list of packages discovered and placed into the catalog
 	ArtifactRelationships []Relationship `json:"artifactRelationships"`
-	Source                Source         `json:"source"`     // Source represents the original object that was cataloged
-	Distro                Distro         `json:"distro"`     // Distro represents the Linux distribution that was detected from the source
-	Descriptor            Descriptor     `json:"descriptor"` // Descriptor is a block containing self-describing information about syft
-	Schema                Schema         `json:"schema"`     // Schema is a block reserved for defining the version for the shape of this JSON document and where to find the schema document to validate the shape
+	Files                 []File         `json:"files,omitempty"`   // note: must have omitempty
```

Comment:
Let's see if we can sync and nail down how/why `Files` should be a sibling field to `Artifacts` -  I think I understand, but it would be good to codify it somewhere so we can be very explicit on why they are different.

Reply:
Are all `File` `Artifact`, but not all `Artifact` `File`?

Reply:
Right now the "artifacts" section is just packages, but in the future it may contain other fields that are sibling fields of "artifacts" today (see https://github.com/anchore/syft/issues/555). Until we tackle #555  it would be consistent to leave `Files` as a sibling of `Artifacts`.